app/controllers/catalog_controller.rb

```diff
@@ -120,6 +120,7 @@ def raw
                              online: { label: I18n.t('um_arclight.advanced_search.available_online'), fq: 'has_online_content_ssim:true' }
                            }
     config.add_facet_field 'repository_sim', label: 'Repository', limit: 10
+    config.add_facet_field 'subarea_sim', label: 'Subarea', limit: 10
```

Comment:
We will also need to add subarea_sim to the facets listed in solrconfig.xml (starting line 266)

Reply:
I'll added subarea_sim to the solrconfig.xml and now I am curious why? lol.

Reply:
I am just assuming that solr will only use fields as facets that are configured as facet fields *shrug*